hooks/post_gen_project.py

```diff
@@ -2,79 +2,87 @@
 import json
 import shutil
 import subprocess
-from python_terraform import *
+
+from python_terraform import Terraform
 
 YES_OPTIONS, NO_OPTIONS = frozenset(["y"]), frozenset(["n", ""])
 PERM_OPTIONS = frozenset(["push", "pull", "admin"])
 
+
 def multiple_choices(question):
     selection = []
-    additional = "Would you like to add another (y/n)? "
+    additional = "Would you like to add another (y/n)? [n]: "
 
-    while True:
-        choice = raw_input(question).lower()
+    multi_q_check = True
+    while multi_q_check:
+        choice = input(question).lower()
```

Comment:
raw_input => input... this is a py2 vs py3 thing... easy to handle though, near the top of the script, after inputs, global defs, etc. put something like this:

```
try:
    input = raw_input
except NameError:
    pass
```

then just use `input` everywhere

Reply:
While this is a python2 vs python3 issue... from my understanding, python2 `input` actually runs `eval()` against what's provided which isn't ideal. It does seem that python3 removed that `eval()` though so this work around should work.

Reply:
yep, in py2, `raw_input` is what we want, but in py3 we just want `input` (`raw_input` is a NameError exception). this try/except overwrites the builtin in py2, and ignores the exception in py3, making `input` the callable we want in both...